src/components/_common/overviewFilter/OverviewFilter.tsx

```diff
@@ -1,72 +1,75 @@
 import { useState } from 'react';
+import classNames from 'classnames';
+import { Heading, Tag } from '@navikt/ds-react';
+
 import { Area } from 'types/areas';
-import styles from './OverviewFilter.module.scss';
+
 import { translator } from '../../../translations';
 import { usePageConfig } from 'store/hooks/usePageConfig';
-import { Heading, Tag } from '@navikt/ds-react';
-import classNames from 'classnames';
+
+import styles from './OverviewFilter.module.scss';
 
 interface OverviewFilterProps {
-    filterUpdateCallback: (filters: Area[]) => void;
+    filterUpdateCallback: (filters: Area) => void;
 }
 
 export const OverviewFilter = ({
     filterUpdateCallback,
 }: OverviewFilterProps) => {
     const filterableAreas = Object.values(Area);
+    const [currentArea, setCurrentArea] = useState<Area>(Area.ALL);
     const { language } = usePageConfig();
-    const [filters, setFilters] = useState<Area[]>([]);
 
     const areaTranslations = translator('areas', language);
+    const overviewTranslations = translator('overview', language);
 
     const handleFilterUpdate = (area: Area) => {
-        if (area === Area.ALL) {
-            setFilters([]);
-            filterUpdateCallback([]);
-            return;
-        }
-        const foundAtPos = filters.findIndex((filter) => filter === area);
-        const updatedFilters =
-            foundAtPos > -1
-                ? filters.filter((filter) => filter !== area)
-                : [...filters, area];
-        setFilters(updatedFilters);
-        filterUpdateCallback(updatedFilters);
+        setCurrentArea(area);
+        filterUpdateCallback(area);
     };
 
     return (
         <div className={styles.overviewFilter}>
-            <Heading size="large" level="3">
-                Velg område
+            <Heading size="medium" level="3">
+                {overviewTranslations('chooseArea')}
             </Heading>
-            <div className={styles.tagWrapper}>
-                {filterableAreas.map((area) => {
-                    const isActive =
-                        filters.includes(area) ||
-                        (area === Area.ALL && filters.length === 0);
+            <nav
+                role="navigation"
+                aria-label={overviewTranslations('ariaExplanation')}
+                className={styles.tagWrapper}
```

Comment:
tagWrapper klassen finnes ikke (skal få inn en eslint-plugin for css-modules, er flere lignende ugyldige imports i koden nå ser jeg)